db/migrate/20220819194804_migrate_roles.rb

```diff
@@ -0,0 +1,21 @@
+class MigrateRoles < ActiveRecord::Migration[7.0]
+  def change
+    User.find_each do |user|
+      if user.organization_id.present?
+        org = Organization.find_by_id(user.organization_id)
+        user.add_role(:org_user, org) if org
```

Comment:
```suggestion
        user.add_role(:user, org) if org
```
I'am wondering if we want to avoid with `org_` in front? I think it is clear enough that a person has the role of user on org.

Reply:
Having `org_` in the name allows us to easily do things like `if user.has_role?(:org_user, :any)`. That makes it more obvious we're talking about a bank user as opposed to a partner user.